google/cloud/pubsub/internal/streaming_subscription_batch_source_test.cc

```diff
@@ -512,7 +514,7 @@ TEST(StreamingSubscriptionBatchSourceTest, AckMany) {
               cq, std::move(context), request);
           using Request = ::google::pubsub::v1::StreamingPullRequest;
           // Add expectations for Write() calls with empty subscriptions, only
-          // the first call has a non-empty value and it is already set.
```

Comment:
While we're tweaking, `s/subscriptions, only/subscriptions. Only/`

Reply:
Done, here and elsewhere.